string ble_status
float64 firstPath_power
float64 aoa
float64 distance
